account_fiscal_agent/models/account_invoice.py

```diff
@@ -8,7 +8,8 @@
 class account_invoice(models.Model):
     _inherit = 'account.invoice'
 
-    agent_invoice_id = fields.Many2one('account.invoice', 'Agent Invoice')
+    agent_invoice_id = fields.Many2one(
+        'account.invoice', 'Agent Invoice', readonly=True)
 
```

Comment:
This is we added,(was not a part of the doc) to create a link between the 2 invoices.